Config.groovy

```diff
@@ -383,3 +383,14 @@ structurizr.with {
     }
 }
 //end::structurizrConfig[]
+
+//tag::openAIConfig[]
+// Configuration for openAI related tasks
+openAI = [:]
+
+openAI.with {
+    //token = ''
+    //model = "text-davinci-003"
+    //maxToken = '500'
```

Comment:
shouldn't we also add the temperature as parameter`?

Reply:
wouldn't that prevent the possibillity to regenerate the result if it is not accurate enough? 0.0 means the user would have to change the given input

Reply:
0.0 means: give me the statistically best answer. I think this would be ok. But I don't mind if we keep 0.3 (or anything else), because I think we have to give it a try.